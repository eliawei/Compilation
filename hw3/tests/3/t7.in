// illegal range of set
void main() {
    set[5..5] myset1;
}
