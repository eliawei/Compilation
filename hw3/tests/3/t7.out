line 3: set range 5 to 5 is invalid
